app/hotels/src/allHotels/NewAllHotelsSearch.js

```diff
@@ -59,7 +59,7 @@ class NewAllHotelsSearch extends React.Component<Props> {
       roomsConfiguration,
       currency,
       orderBy,
-      filterParams,
+      filterParams: { hotelAmenities, ...filterParmsRest },
```

Comment:
`filterParmsRest`, it looks like there's a type in the variable name.

Reply:
If you mean typo, you are correct, I will fix it

Reply:
Yes, I meant typo 😃